brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1047 Fix: Settings Page - Height issue on setting page for Copy Paste Styles.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1047

### Screenshots

- Issue - https://share.getcloudapp.com/2NumXrmW
- Fixed - https://share.getcloudapp.com/geuRW2wZ

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go to the Spectra Settings page.
- Go to the Editor Options > Check Copy Paste Styles spacing.

### Checklist:
